routers/web/repo/actions/badge.go

```diff
@@ -19,13 +19,33 @@ import (
 func GetWorkflowBadge(ctx *context.Context) {
 	workflowFile := ctx.PathParam("workflow_name")
 	branch := ctx.Req.URL.Query().Get("branch")
-	if branch == "" {
+	tag := ctx.Req.URL.Query().Get("tag")
+	useLatestTag := ctx.Req.URL.Query().Has("latest_tag")
+	var ref string
+	switch {
+	case useLatestTag:
+		tags, _, err := ctx.Repo.GitRepo.GetTagInfos(0, 1)
```

Comment:
I think you should use a function like `GetLatestReleaseByRepoID` to do that.

Reply:
So we need the tag of the latest release here? Do we need to rename it to `latest_release_tag`?

Reply:
No. All tags have been synced into databases. They shared the same table with releases.